app.js

```diff
@@ -6,15 +6,34 @@ const options = require("./utils/options");
 const geocode = require("./utils/geocode");
 const forecast = require("./utils/forecast");
 const menu = require("./utils/menu");
+const request = require("request");
+
+////////////////////////
+///////////////////////
```

Comment:
unused module. If you need to make a request, use `postman-request` as request module is deprecated. postman-request is a fork of the same module

Reply:
ok sir